proposals/sandbox/krustlet.md

```diff
@@ -0,0 +1,91 @@
+# Krustlet
+
+## Description
+
+Krustlet is a Kubernetes Kubelet implementation for running WebAssembly (Wasm) modules on nodes.
+Krustlet acts as a Kubelet by listening on the event stream for new pods that the scheduler assigns
+to it based on specific Kubernetes tolerations. It also can be extended to enable other types of
+non-Wasm runtimes using a "Provider Model" similar to [Virtual
+Kubelet](https://github.com/virtual-kubelet/virtual-kubelet), but its primary use case is for Wasm.
+
+## Project Goal
+
+The primary goal of Krustlet is to be the easiest way to integrate Wasm support as seamlessly as
+possible into Kubernetes. It will allow for users to run heterogeneous clusters where both
+containers and Wasm modules can run (though not necessarily on the same node)
+
+## Current Status
+
+Krustlet is very close to a 1.0 release. It was created in late 2019 and has grown along with the
+rest of the Wasm community. Essentially 1.0 is stating that that basic features (but not all of
+them) for a kubelet are in place and the software is reliable enough to use in production. Continued
```

Comment:
Minor clarification needed here: "but not all of..." what? All basic features, all features?

Reply:
Ok I think I clarified it